DP3TApp/Logic/Config/ConfigManager.swift

```diff
@@ -77,7 +80,12 @@ class ConfigManager: NSObject {
         }
     }
 
-    private func shouldLoadConfig(backgroundTask: Bool) -> Bool {
+    private func shouldLoadConfig(backgroundTask: Bool, url: String?) -> Bool {
+        // if the config url was changes (by OS version or app version changing) load config
+        if Self.lastConfigUrl != url {
```

Comment:
This will return true if both are nil, is that the intended behaviour? this will override the lines below where you check for background

Reply:
no, i changed the behavior.